tyrian/shared/src/main/scala/tyrian/Attr.scala

```diff
@@ -4,6 +4,16 @@ package tyrian
 sealed trait Attr[+M]:
   def map[N](f: M => N): Attr[N]
 
+/** An attribute of an HTML tag that is does not exist, used as a "do not render" placeholder
```

Comment:
```suggestion
/** An attribute of an HTML tag that does not exist, used as a "do not render" placeholder
```
? not sure if that was the intended meaning